astroid/nodes/scoped_nodes/scoped_nodes.py

```diff
@@ -2139,9 +2139,10 @@ def _newstyle_impl(self, context: InferenceContext | None = None):
     @cached_property
     def fromlineno(self) -> int | None:
         """The first line that this node appears on in the source code."""
-        if not PY38_PLUS or PY38 and IS_PYPY:
+        if not PY38_PLUS or IS_PYPY and PY38 and not PYPY_7_3_11_PLUS:
```

Comment:
Do we want to support multiple PyPy versions? I guess it's simple enough here, although I wouldn't add any more to CI.

Reply:
This feel like a lot of work but also added values. I guess on a case by case basis... ?

Reply:
> This feel like a lot of work but also added values. I guess on a case by case basis... ?

Yeah, I was more referring to this specific case 😄